build-tools/build-indexes

```diff
@@ -9,7 +9,7 @@ const rootDir = path.resolve(__dirname, '..');
 process.chdir(rootDir);
 
 if (!fs.existsSync('data/Pokemon-Showdown')) {
-	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git', {
+	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git Pokemon-Showdown', {
```

Comment:
It would break existing installs on case-sensitive operating systems, but I think I'd prefer for the directory to be lowercase going forward.

Reply:
OK.